implants/lib/c2/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "c2"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+tonic = { workspace = true }
+prost = "0.12"
```

Comment:
Can we move this to the parent workspace.
If other parts of realm use it in the future would be nice to continue our standard of using one version of a dep across the project.

Reply:
Sure, but nothing else is using it right now